distribution/scripts/ballerina/service_chaining/main.bal

```diff
@@ -0,0 +1,57 @@
+import ballerina/http;
+
+type Person record {|
+    string name;
+    int age;
+|};
+
+type Response record {|
+    json w1;
+    json w2;
+|};
+
+configurable string epKeyPath = ?;
+configurable string epKeyPassword = ?;
+
+listener http:Listener securedEP = new (9090,
+    secureSocket = {
+        key: {
+            path: epKeyPath,
+            password: epKeyPassword
+        }
+    }
+);
+
+final http:Client nettyEP = check new ("https://netty:8688",
+    secureSocket = {
+        cert: {
+            path: epKeyPath,
+            password: epKeyPassword
+        },
+        verifyHostName: false
+    }
+);
+
+function convert(json|error data) returns json {
+    if data is error {
+        return data.message().toJson();
+    }
+    return data;
+}
+
+service /serviceChaining on securedEP {
+    resource function post .(@http:Payload json payload) returns json {
```

Comment:
I don't think this resource function isolated since both of the workers are accessing the payload. I think making the payload readonly should resolve the issue
```suggestion
    resource function post .(@http:Payload readonly & json payload) returns json {
```